### PR TITLE
Examples JSDoc tokenizer fixes

### PR DESCRIPTION
**Pull request by kpal81xd**
- JSDoc tokenizer fixes
- Inline type casting for canvas decl

### CODE REVIEW

File: examples/README.md [hunk 1]
```diff
@@ -45,10 +45,8 @@ Each example consists of two modules to define its behavior:
 ```js
 import * as pc from 'playcanvas';
 
-const canvas = document.getElementById('application-canvas');
-if (!(canvas instanceof HTMLCanvasElement)) {
-    throw new Error('No canvas found');
-}
+const canvas = /** @type {HTMLCanvasElement} */ (document.getElementById('application-canvas'));
```
Comment (willeastcott) [inline examples/README.md:48]: Why the outer brackets?

Comment (kpal81xd) [reply examples/README.md:48]: Mantadory for jsdoc type casting

Comment (kpal81xd) [reply examples/README.md:48]: https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#casts

Comment (willeastcott) [reply examples/README.md:48]: Can you put the JSDoc on the line above?

File: examples/src/examples/animation/blend-trees-1d.example.mjs [hunk 2]
```diff
@@ -2,10 +2,8 @@ import * as pc from 'playcanvas';
 import { data } from 'examples/observer';
 import { deviceType, rootPath } from 'examples/utils';
 
-const canvas = document.getElementById('application-canvas');
-if (!(canvas instanceof HTMLCanvasElement)) {
-    throw new Error('No canvas found');
-}
+const canvas = /** @type {HTMLCanvasElement} */ (document.getElementById('application-canvas'));
```
Comment (kungfooman) [inline examples/src/examples/animation/blend-trees-1d.example.mjs:5]: Casting is not knowing... fake type safety that breaks at any refactor

File: examples/README.md (hunk 1 above)
Comment (willeastcott) [reply examples/README.md:48]: OK, I take it back. You can't. 👍

Comment (kpal81xd) [reply examples/README.md:48]: No casting can only be inline since the jsdoc type is refering to the rhs of the expression not the left

**Review by willeastcott — APPROVED** [review]

File: examples/src/examples/animation/blend-trees-1d.example.mjs (hunk 2 above)
Comment (kpal81xd) [reply examples/src/examples/animation/blend-trees-1d.example.mjs:5]: application-canvas is hard coded into the html file so its guaranteed.

**Pull request merged**